iaas/20221109.md

```diff
@@ -0,0 +1,64 @@
+# 2022-11-09
+## Participants - use of github handle is preferred
+* @fkr
+* @tibeer
+* @matfechner
+* @ra-beer
+* @master-caster
+* @o-otte
+* @garloff
+* @osfrickler
+* @ppkuschy
+
+## Recurring items
+* Who is presenting in weekly community call and transforming etherpad to minutes in GitHub?
+  * @o-otte
+* Anything else to add to the agenda?
+
+* Housekeeping
+
+## Agenda
+
+### Sprint Review
+
+#### https://github.com/SovereignCloudStack/issues/issues/91
+Done except for daily jobs -> create new issue for this
+
+#### https://github.com/SovereignCloudStack/issues/issues/168
+Changes for this cycle done, i.e. https://release.osism.tech/notes/5.0.0.html#openstack-image-manager
+@tibeer will get Infos what is done exactly
+
+Documentation topic: https://github.com/osism/openstack-image-manager/issues/336
+To be done in three weeks from today.
+
+#### Cloud-in-a-Box
+https://github.com/SovereignCloudStack/issues/issues/130 still needed
+
+node-image and installer manager differ in versions. @master-caster will also do tests.
+Images needs evaluation -> https://github.com/SovereignCloudStack/issues/issues/216
+
+#### Zuul for SCS Requirements
+Discussion about requirements. Two offers are present, one is still needed. -> done
+
+#### Support Air Gap installations & operations and builds https://github.com/SovereignCloudStack/issues/issues/90
+Current work: https://github.com/osism/issues/issues/348 still needs 2 weeks
+
+### Sprint Planning
+* Paul needs refined tasks
+  maybe https://github.com/osism/issues/issues/343 also
```

Comment:
there something is missing: I fixed it in the minutes in the pad. Please sync. Sorry for that.
(beware: i started filling the pad for next week, so just grab the part about the refined tasks for paul).

thanks.

Reply:
I've added the tasks